services/workflows-service/src/collection-flow/collection-flow.service.ts

```diff
@@ -93,12 +95,12 @@ export class CollectionFlowService {
   async updateFlowConfiguration(
     configurationId: string,
     steps: FlowStepModel[],
-    projectIds: TProjectIds,
+    projectId: TProjectId,
   ): Promise<FlowConfigurationModel> {
     const definition = await this.workflowDefinitionRepository.findById(
       configurationId,
       {},
-      projectIds,
+      projectId ? [projectId] : null,
```

Comment:
weird? why is it like that?
it should be received as projectIds and other project ID.
as you can have multiple projects on same user.
but single project id of definition